packages/material-ui/src/InputBase/InputBase.test.js

```diff
@@ -38,6 +38,7 @@ describe('<InputBase />', () => {
     classes,
     inheritComponent: 'div',
     mount,
+    propsSpread: false,
```

Comment:
That would be a breaking change. I don't understand how this relates to passing through of aria attributes.

Reply:
https://github.com/mui-org/material-ui/pull/14580#pullrequestreview-205310201

Reply:
But that's not what I said. I wanted to change to root component from the `div` to the `input`.

Reply:
You have introduced a new concept: a top component. So, you would like to keep the current changes but to push them one step further? root -> container, input -> root?

Reply:
A few components don't have any top component, like the Tooltip.